tensorflow/java/src/main/java/org/tensorflow/Session.java

```diff
@@ -81,6 +81,19 @@ public Session(Graph g, byte[] config) {
     graphRef = g.ref();
   }
 
+  /** wrap native method */
+  public long makeCallable(byte[] config){
+    return makeCallable(this.nativeHandle, config);
+  }
```

Comment:
I guess the allocated "callable" is a resource which lifetime should be managed by the JVM, e.g. using `try-with-resources`, unless it can only be tied to the `Session` instance itself?